### ISSUE TITLE
order is not maintained

### ISSUE THREAD

**Issue by gperdomor** [xavierfaucon/bootstrap-selectsplitter#3]
The order of the options is not the same of the original select...

For example, change the example code with

``` html
<div class="container">
    <div class="row">
        <div class="col-xs-12" style="padding-top:40px;">
            <select data-selectsplitter-selector>
              <optgroup label="Category 1">
                <option value="5">Choice 1</option>
                <option value="3">Choice 2</option>
                <option value="2">Choice 3</option>
                <option value="4">Choice 4</option>
              </optgroup>
              <optgroup label="Category 2">
                <option value="5">Choice 5</option>
                <option value="6">Choice 6</option>
                <option value="7">Choice 7</option>
                <option value="8">Choice 8</option>
              </optgroup>
              <optgroup label="Category 3">
                <option value="5">Choice 9</option>
                <option value="6">Choice 10</option>
                <option value="7">Choice 11</option>
                <option value="8">Choice 12</option>
              </optgroup>
            </select>
        </div>
    </div>
</div>
```

and the options shown when you click on Category 1, are Choice 3, Choice 2, Choice 4, Choice 1 (in this order), instead Choice 1, Choice 2, Choice 3 and Choice 4

**Comment by X13454** [comment]
Hello,

I'll look into that tomorrow :)

**Comment by X13454** [comment]
Your issue should be corrected with version 0.1.2. 
The option elements were displayed in the order the jQuery .find (https://api.jquery.com/find/)  would find them.

Now I used jQuery index to sort them (https://api.jquery.com/index/).
